CalculatorSwitch/CalculatorSwitch.java

```diff
@@ -0,0 +1,49 @@
+import java.util.Scanner;
+public class CalculatorSwitch {
+    public static void main(String[] args) {
+        double firstNumber;
+        double secondNumber;
+        char operator = ' ';
+        double result;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Hello. Please, enter first number");
+        try {
```

Comment:
Please do initializing of firstNumber and secondNumber in one place (and one try...catch)